back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -20,12 +20,34 @@ def index
       .order('root_ordering, admin_publications.ordering')
 
     @publications = paginate publications
-    @publications = @publications.includes(:publication, :children)
 
-    included = if params[:include_publications] == 'true'
-      %i[publication publication.avatars publication.project_images publication.images publication.current_phase]
+    included = []
+
+    # Disambiguation: @publications is a collection of AdminPublication records, whereas :publication in the includes()
+    # method, and/or the `included` array passed to the serializer, refers to the associated project or folder.
```

Comment:
If this is the case then it would probably be worth changing the variable names to `@admin_publications` will make it easier to follow